_episodes/01-introduction.md

```diff
@@ -16,17 +18,31 @@ keypoints:
 - "CWL supports software containers, supporting reproducibility on different machines"
 ---
 
-# Common Workflow Language
+# Computational Workflows
 
 Computational workflows are widely used for data analysis, enabling rapid innovation and decision making.
-_Workflow thinking_ is a form of "conceptualizing processes as recipes and protocols, structured as workflow or dataflow graphs with computational steps,
-and subsequently developing tools and approaches for formalizing, analyzing and communicating these process descriptions" ([Gryk & Ludascher, 2017](https://doi.org/10.1353/lib.2017.0018)).
+_Workflow thinking_ is a form of "conceptualizing processes as recipes and protocols, structured as workflow or dataflow graphs with computational steps, and subsequently developing tools and approaches for formalizing, analyzing and communicating these process descriptions"[^1].
```

Comment:
Most of these lines would be picked up as too long by `make lesson-check-all`. Some line breaks would be good.

Reply:
I've added some more linebreaks. And added some extra long lines for the table - but unfortunately that doesn't seem to cope with linebreaks as well.